lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -210,7 +213,13 @@ private List<VcStatusDto> generateVcSuccessStatuses(List<SignedJWT> credentials)
 
             CredentialIssuerConfig addressCriConfig =
                     configurationService.getCredentialIssuer(addressCriId);
-            boolean isSuccessful = VcHelper.isSuccessfulVc(signedJWT, addressCriConfig, true);
+            Map<String, ContraIndicatorScores> ciScoresMap =
+                    configurationService.getContraIndicatorScoresMap();
+            int ciScoreThreshold =
+                    Integer.parseInt(configurationService.getSsmParameter(CI_SCORING_THRESHOLD));
+            boolean isSuccessful =
+                    VcHelper.isSuccessfulVc(
+                            signedJWT, addressCriConfig, ciScoresMap, ciScoreThreshold);
```

Comment:
I don't think we should do the CI scoring here. It should be at the top level across all VC's gathered. Not against an individual VC

Reply:
Yes, agreed, the scoring is not required here.